[2022-07-19 13:41:01,431] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: simple_dag.say_hi scheduled__2022-07-19T13:40:00+00:00 [queued]>
[2022-07-19 13:41:01,450] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: simple_dag.say_hi scheduled__2022-07-19T13:40:00+00:00 [queued]>
[2022-07-19 13:41:01,451] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-19 13:41:01,451] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-19 13:41:01,453] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-19 13:41:01,484] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): say_hi> on 2022-07-19 13:40:00+00:00
[2022-07-19 13:41:01,515] {standard_task_runner.py:52} INFO - Started process 403 to run task
[2022-07-19 13:41:01,522] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'simple_dag', 'say_hi', 'scheduled__2022-07-19T13:40:00+00:00', '--job-id', '766', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpcy8qv49u', '--error-file', '/tmp/tmpx1fqes1e']
[2022-07-19 13:41:01,523] {standard_task_runner.py:80} INFO - Job 766: Subtask say_hi
[2022-07-19 13:41:01,715] {task_command.py:370} INFO - Running <TaskInstance: simple_dag.say_hi scheduled__2022-07-19T13:40:00+00:00 [running]> on host 1ead164eba69
[2022-07-19 13:41:01,894] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=simple_dag
AIRFLOW_CTX_TASK_ID=say_hi
AIRFLOW_CTX_EXECUTION_DATE=2022-07-19T13:40:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-19T13:40:00+00:00
[2022-07-19 13:41:01,898] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-07-19 13:41:01,900] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "Hi!!"']
[2022-07-19 13:41:01,920] {subprocess.py:85} INFO - Output:
[2022-07-19 13:41:01,924] {subprocess.py:92} INFO - Hi!!
[2022-07-19 13:41:01,925] {subprocess.py:96} INFO - Command exited with return code 0
[2022-07-19 13:41:01,993] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=simple_dag, task_id=say_hi, execution_date=20220719T134000, start_date=20220719T134101, end_date=20220719T134101
[2022-07-19 13:41:02,057] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-19 13:41:02,140] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
